bpf/lib/common.h

```diff
@@ -482,6 +484,7 @@ enum metric_dir {
  *    In the IPsec case this becomes the SPI on the wire.
  */
 #define MARK_MAGIC_HOST_MASK		0x0F00
+#define MARK_MAGIC_PROXY_EGRESS_EPID	0x0900 /* mark carries source endpoint ID */
```

Comment:
Can you also update the text above to highlight that depending on this field, the upper 16 bits hold endpoint ID instead?

Reply:
Done.